scripts/apps/master-desk/components/OverviewComponent.tsx

```diff
@@ -0,0 +1,220 @@
+                                    {(this.props.stages?.[desk._id] ?? []).map((stage, i) => (
+                                        this.getStageItems(stage).length ? (
+                                            <React.Fragment key={i}>
+                                                <div className="sd-board__subheader">
+                                                    <h5 className="sd-board__subheader-title">{stage.name}</h5>
+                                                </div>
+                                                <ul className="sd-list-item-group sd-shadow--z2 inline-content-items">
+                                                    {this.getStageItems(stage).map((item, key) => (
+                                                        <li className="content-item" key={key}>
+                                                            <div className="content-item__type">
+                                                                <TypeIcon
+                                                                    type={item.type}
+                                                                    highlight={item.highlight}
+                                                                />
+                                                            </div>
+                                                            <div className="content-item__urgency-field">
+                                                                <ItemUrgency item={item} />
+                                                            </div>
+                                                            <div className="content-item__text">
+                                                                <span className="keywords">{item.slugline}</span>
+                                                                <span id="title" className="headline">
+                                                                    {item.headline}
+                                                                </span>
+                                                            </div>
+                                                            <div className="content-item__date">
+                                                                <TimeElem date={item.versioncreated} />
+                                                            </div>
+                                                        </li>
+                                                    ),
+                                                    )}
+                                                </ul>
+                                            </React.Fragment>
+                                        ) : null
+                                    ))}
+                                </div>
+                            </div>
+                        ) : null,
+                    )}
+                </div>
+            ) : (
```

Comment:
List view is activated when you apply filters. Then it is converted to list view.

Reply:
Right. The naming is not the best here, because it's not only the view type that is different here, but the data that's displayed. In one case, desks are displayed and in other a list of articles. It'd be good to update the naming so it's more clear.